src/dropdown/Dropdown.js

```diff
@@ -199,9 +194,25 @@ export default class Dropdown extends React.Component {
   }) => {
 
     const Option = optionComponent;
-    const propertyName = this.props.groupByKey;
+    const { groupByKey } = this.props;
+    const groupedOptions = options.reduce((acc, o, i) => {
+      const shouldCreateNewGroup = i === 0 || o[groupByKey] !== options[i - 1][groupByKey];
+      if (shouldCreateNewGroup) {
+        const newGroup = {
+          optionGroupTitle: o[groupByKey] || '',
```

Comment:
right...
still, we are passing `""` when there is no title: shouldn't that logic be updated?